### PR TITLE
Removes xenolings

### PR DESCRIPTION
**Pull request by ghost**
but honestly immortal crewmembers who can fight other lings and absorb them to permanently kill them and spy on them is stupid and it's not worth the amount of balancing you have to do to remove every little thing about it

and now to address all possible arguments

'well instead of that, why don't you-'
no
#### Changelog

:cl:  
rscdel: Removed xenobiologists making themselves changelings
/:cl:

### CODE REVIEW

**Comment by Amelia0010** [comment]
ok

**Comment by Hopekz** [comment]
Disclamer:  This is not a meme PR . Please don't upvote/downvote for the lulz it is something that is actually being discussed.

**Comment by MonsWTF** [comment]
Liberate XenoBio, game-play evolution of our times.

**Comment by Hopekz** [comment]
Xeno is not as big of a deal as people make it out to be. 
In order to be "effective" in xeno you need 40-50 uninterrupted minutes to set up any of the scenarios this is trying to counter. Xeno does not get a chance to make a big difference in shifts I played for the last 2-3 years (on and off). 
Off the top of my head I can think of a single shift where it mattered to a traitor xeno scientist but even him using this system did not ruin the fun for others, it just gave them a greentext.

Unsure of why this is being taken seriously. This change is going to remove a big chunk of features already built to fix a much smaller issue (assuming one even realistically exists at the moment).

I do not play xeno so I have an outside perspective of someone who does not use this system. Xeno being effective and unstoppable is not the case from my observations.

Xeno does have a big anti ling validhunting tool which is being addressed in another PR once it gets merged (pheromone receptors being removed from xenolings)

**Comment by CMOisLing** [comment]
Ow no what will Xenobio do now?

**Comment by Ratvar** [comment]
It takes **20** minutes, and xenoling still gets to hear ling radio to validhunt them.  It is too powerful of a tool, and time doesn't balance anything.
So, imagine if R&D got full traitor uplink at 60 minutes. They always buy traitor radio and check it just in case some antags try to cooperate beyond murderboning, and they also get all other weapons.
After years of abuse, esword and revolver get cut out out of science uplink, leaving adrenals, EMP grenades, spacesuit, holoparasites and all other crap in. That's what xenoling is.
The whole "I collect all powers for MYSELF" is robotics making a durand for himself level of shitlery, but this time there's not even a resource/techweb resource investment. 
That xenoling cannot be used in any situation but validhunting/antagging is too an issue, and when it's flexed it causes security to either fuck your ass, which's a problem, or to join your side, which empowers security massively fnr and fucks round as much as if they let changeling hunt antags for them and it did.
Every time a tool from it was removed, xenobio mains whined that "we were afk in a reinforced box interacting for nobody so long that we need to get all antag powers, it's not even an issue because nobody but us powergamers who like to afk play xenobio". It should stop, and they should get a better power instead.

**Comment by ghost** [comment]
@Hopekz 
It is exactly that big a deal.

Don't give feedback on this, xeno mains aren't welcome

Though I mean if we want to pretend to answer your idiotic comment let's summarize.

>Xeno is not as big of a deal as people make it out to be.

Yes it is. You have timestop, instant heals, changelingification, etc.

>In order to be "effective" in xeno you need 40-50 uninterrupted minutes to set up any of the scenarios this is trying to counter. 

You need less than that.

>Xeno does not get a chance to make a big difference in shifts I played for the last 2-3 years (on and off).

You clearly have not been playing for those 2-3 years.

>Off the top of my head I can think of a single shift where it mattered to a traitor xeno scientist but even him using this system did not ruin the fun for others, it just gave them a greentext.

Absolute nonsense.

>Unsure of why this is being taken seriously. This change is going to remove a big chunk of features already built to fix a much smaller issue (assuming one even realistically exists at the moment).

The 'features' resolve nothing. Changeling nonantags are problematic from both a rules perspective and a balance perspective because everything must take into account that the guy using it is potentially immortal.

>I do not play xeno

Evidently.

>so I have an outside perspective of someone who does not use this system. Xeno being effective and unstoppable is not the case from my observations.

You have obviously never watched a single game.

>Xeno does have a big anti ling validhunting tool which is being addressed in another PR once it gets merged (pheromone receptors being removed from xenolings)

This is hardly the only issue they have.

With a minimum of effort, xenobiologists can become immortal and get free movespeed buff, adrenals, thermals, total immortality unless cremated or gibbed, the ability to swap species on the fly, rapid damage healing, and private lingchat to eavesdrop on other changelings and rat them out.

For comparison, this would be like if the Chaplain could craft stun talismans, create constructs independently, and draw teleport runes after 40 minutes. Or if R&D could print adrenal implants and energy swords.

I shouldn't need to elaborate on why this is stupid and why 'uhh it..it tak TIMe..its...its take time...' is an idiotic comment.

**Comment by Hopekz** [comment]
> @Hopekz
> It is exactly that big a deal.

Okay, Adam. I guess we both have a subjective opinion. 
I am now going to answer your inquiries similar to your style. Line by line so I don't miss any of your strong arguments.

> Don't give feedback on this, xeno mains aren't welcome

Never have I even done xeno even once. Nice try bully. 
My perspective is 100% outside of xeno as I have never played that role. 
I have turned into a xenoling a few times however making me just as much of an expert as you.

> Though I mean if we want to pretend to answer your idiotic comment let's summarize.

Oh boy here we go. You know a counter argument is going to be good when you need to talk down on a different point of view in this manner. I too find your stance idiotic. This is the whole reason we're exchanging right now.

> > Xeno is not as big of a deal as people make it out to be.

Exactly my point. I do not think this is as big of a deal as it is made out to be. I am on the side of this feature not being removed because I think people are blowing it out of proportion.

> Yes it is. You have timestop, instant heals, changelingification, etc.

I thought this PR is targeting only xenolings. If you're trying to address all of these things with this PR we should maybe discuss xeno balance first before we make a decision.

> > In order to be "effective" in xeno you need 40-50 uninterrupted minutes to set up any of the scenarios this is trying to counter.
> 
> You need less than that.

Sorry, Adam, I'm not sitting here with a stopwatch man I'm just saying its a range between that time.
When I give you the time range its more of me mentally filtering only the rounds where it actually made a difference. 
I get you can speed run a department the same way you can speedrun botany or robo to get your results.

> > Xeno does not get a chance to make a big difference in shifts I played for the last 2-3 years (on and off).
>
> You clearly have not been playing for those 2-3 years.

I thought I was! Foiled my argument again!
Someone having a different opinion than you must mean they just haven't played the game as much! 
(I'm kidding but look how subjectively stupid I sounded enforcing that argument).

> > Off the top of my head I can think of a single shift where it mattered to a traitor xeno scientist but even him using this system did not ruin the fun for others, it just gave them a greentext.
> 
> Absolute nonsense.

Thank you, Adam.

> > Unsure of why this is being taken seriously. This change is going to remove a big chunk of features already built to fix a much smaller issue (assuming one even realistically exists at the moment).
> 
> The 'features' resolve nothing. Changeling nonantags are problematic from both a rules perspective and a balance perspective because everything must take into account that the guy using it is potentially immortal.

Why is taking down a non-antag that big of a deal? You're not supposed to be taking them down as crew anyway because they are not a threat. Are we dealing with more of a self antag issue with these changes? At first, I thought you meant they are too powerful for the crew to take down as antags but now you are shifting it over to non-antags.

> > I do not play xeno
> 
> Evidently.

Thank you. I was trying to convey that in my previous points.
My perspective is 100% on the outside of these benefits. I am from the perspective of the individuals that face these xenoling effects that you are saying are a big deal.

> > so I have an outside perspective of someone who does not use this system. Xeno being effective and unstoppable is not the case from my observations.
> 
> You have obviously never watched a single game.

Hard to take these remarks seriously. Moving on.

> > Xeno does have a big anti ling validhunting tool which is being addressed in another PR once it gets merged (pheromone receptors being removed from xenolings)
> 
> This is hardly the only issue they have.

Please elaborate. 
I'd rather everything be in the open so we can really dig into what the issue is.
Your reasoning for the removal of this feature, in my opinion, is weak.


> With a minimum of effort, xenobiologists can become immortal and get free movespeed buff, adrenals, thermals, total immortality unless cremated or gibbed, the ability to swap species on the fly, rapid damage healing, and private lingchat to eavesdrop on other changelings and rat them out.

From the perspective of being the crew, I do not see any of these being bad except the eavesdropping part. 
I am in favor of removing the valid tools of pheromone receptors (being addressed in open PR) and ling chat. 
Even if an antagonist gains these abilities you are just fighting another ling as sec.

> For comparison, this would be like if the Chaplain could craft stun talismans, create constructs independently, and draw teleport runes after 40 minutes. Or if R&D could print adrenal implants and energy swords.

While I do like your comparison, I do not like the job you used. You specifically chose a job that has a lenient valid hunting role. 
Non-antagonist scientists should NOT be valid hunting per our server rules.
Any other random job should not be valid hunting as well. We ban on our server for breaking this rule. (see Serp's ban [first one off the top of my head] )

> I shouldn't need to elaborate on why this is stupid 

Thank you, Adam.
You do need to elaborate. 
Not everyone has your point of view man we're here to exchange those views.

> and why 'uhh it..it tak TIMe..its...its take time...' is an idiotic comment.

In my personal opinion the time argument is valid and here is why (please feel free to debunk me on this as that's why I'm exchanging):
Yogs shifts are not that long time-wise. 
I think an antagonist should have the ability to prepare a solution to greentext when dedicating 40 minutes of his time to that solution. There is a big period of time where something can be done to stop him from doing so. There is even a period of time where the xenobiotic is immobile. Why would you remove the Durand from the game just because robotics can take the time to build it? (Using my time argument) 
In robotics, for example, there is plenty that can be done to stop it. There is even a period of time where the roboticist is vulnerable with a big durand shell in his office. This is very similar to the period of complete vulnerability that the xenobiologist experiences in my opinion.

**Comment by oklews** [comment]
I love you Adam, I am on the bandwagon of improve not remove for most things but benobio lings are just annoying and unfair

**Comment by deadmomo69** [comment]
usually i agree with improve, not remove, but at least this stops all the arguments about balancing it and nerfing it until all we have is literally just a flesh-suit as a xenoling
i never understood how people in xenobio could become lings anyway, since i doubt they'd be connected to the hivemind whatsoever.

**Comment by MegaEmpirical** [comment]
I can definitely get behind this.

And yeah, normally I'd say improve instead but this isn't really that kind of situation.

**Comment by Szyszkrzyneczka** [comment]
once i saw a xenobiology man get all the buffs, become a xling and proceed to break inside CMO office, he was spotted by HoS who was running away the second he saw he couldn't do much against such autism. Merge please

**Comment by ghost** [comment]
God I think I hate this guy more than I hate dareb.

@Hopekz 

Don't pretend we're having a discussion where we both try to convince each other of the validity of our respective arguments. You don't have a leg to stand on and this is an argument where I relentlessly insult you until you leave.

Xenobiology is still absolutely as big a deal as people make it out to be. Don't try to take things piecemeal out of xenobiology-either we address it whole, or not at all. We aren't 'just' addressing xenolings. We're addressing xenobiology balance, of which xenolings are a part.

Would xenolings, on their own, be balanced? Probably not, but it would leave room for argument. However, xenolings with access to mob armies, the ability to turn into rare golem species, timestop, and instant adminheal cores are a different matter entirely, which is why we brought it up. Don't try to separate it like 'well this PR balances xenolings' because everything else is why this is a discussion at all.

Robotics and especially botany also need very little time to become problem children. But unless your shift ends in thirty minutes or less (IE: Wizard) xenobiologists usually have enough time to field what they want from their department.

Xenoling needs merely a gold slime and a rainbow slime. This is achievable in twenty minutes. A xenobiologist can become a changeling in less time than it takes for war ops to even arrive.

There's ample evidence for shifts where it DID matter that the xenobiologist became a changeling. Just look on the forums. For christ sake, I can't even play and even I can see that. Mountain after mountains of shifts where the xenobiologist ling killed the other lings/a traitor/a cult. Any round that goes on for longer than 40 minutes is virtually guaranteed to have a friendly crewmember running around who is immune to any and all damage, and more than likely that's the Captain if they get wind of xenobiology doing that and turn themselves into one.

It merely rewards powergaming. There isn't any reason to have it, not even as a reward.

'From the perspective of being the crew, I don't see why any of these are bad'

Really? You don't see why on the fly, no-downsides stun immunity, rapid damage healing, and literal immunity to being killed is bad?

**Comment by MegaEmpirical** [comment]
I'm seeing xenolings more and more often.

I'm seeing them valid hunt during late night low pop and letting them get away with it because it's low pop and no security. I hate it. 

I saw one round with a 3 xenolings on low pop. Captain, scientist, and some other.

**Comment by drderp3635** [comment]
>xenoling in 20 minutes

Sure bud

**Comment by Szyszkrzyneczka** [comment]
It is possible to pull that shit faster

**Comment by drderp3635** [comment]
When you look at the benefits of xenoling, it is about the same as the benefits you gain from nanites (anti stun, fast healing, granted it doesn't have the revive, but it does have armor) but nobody ever complains about nanites.

Having a way to kill changelings isn't something new or huge, and having xenolings validhunting regular lings is something that should delt with on an administrative level for powergaming, and the pheremone receptor ability has already been removed. 

All in all xenoling wasn't even one of the stronger tools for xeno since it already has had its stronger abilities removed and there could be much better changes made to balance xeno rather than removing it

**Comment by Szyszkrzyneczka** [comment]
In order for nanites to do that shit they must spend research points on it, also they can be just nullified with usage of EMP which unfortunately became a balancing meme just like tf2's "No random crits"

**Comment by CMOisLing** [comment]
Also nanites have the massive weakness of needing a cloud server to be reliable. Because without one they are basically made deadly by an emp or two

**Comment by MonsWTF** [comment]
> Xenoling needs merely a gold slime and a rainbow slime. This is achievable in twenty minutes. A xenobiologist can become a changeling in less time than it takes for war ops to even arrive.

I'm not commenting on much in this thread, but that specifically is horseshit. Give me a scientist that can consistently make Rainbows 20 minutes into the round and I'll surrender ever playing Xenobio again.

Rainbows at ultra-tier RnG and hard-focused by the Xenobio can be achieved at 30 minutes earliest. The average is 40m and the RnG variable range is 35-50m.

**Comment by Ratvar** [comment]
Let's forget red slimes rising mutation chance, making rainbow slime a guaranteed happenstance instead of being based on any sort of RNG. Like, **that** is a horseshit.

**Comment by alexkar598** [comment]
@drderp3635 Please do not wish the death of other contributors as toxic as they are.

**Comment by alexkar598** [comment]
@AdamElTablawy Don't try to prevent players from giving their opinions on a PR despite them being a xeno main or a clown main

File: code/modules/mob/living/simple_animal/hostile/headcrab.dm [hunk 1]
```diff
@@ -22,7 +22,7 @@
 	ventcrawler = VENTCRAWLER_ALWAYS
 	var/datum/mind/origin
 	var/egg_lain = 0
-	gold_core_spawnable = HOSTILE_SPAWN //yogs
+	gold_core_spawnable = NO_SPAWN //yogs
```
Comment (alexkar598) [inline code/modules/mob/living/simple_animal/hostile/headcrab.dm:25]: This leaves a ton of unused code in the codebase, get rid of all the xenoling framework.

**Review by alexkar598 — CHANGES_REQUESTED** [review]

**Comment by Hopekz** [comment]
> God I think I hate this guy more than I hate dareb.

Thank you, Adam.
Not sure if your God will read your reply but nonetheless **I am honored**.

> @Hopekz
> 
> Don't pretend we're having a discussion where we both try to convince each other of the validity of our respective arguments. 

This isn't your first PR. **The moment you introduced this PR is the moment you opened your proposed changes up for debate.**

>You don't have a leg to stand on and this is an argument where I relentlessly insult you until you leave.

**You have a weak argument.** 
Insulting me will not make your argument any stronger.

**Here is why I think your argument is weak:**
You are proposing to remove a feature from our game based on the possibility of it being used in a manner that is not in accordance to our server rules. Your argument relies on the offending individual to go out and valid hunt as a role that should not be valid hunting.
To reinforce my previous point: We punish individuals who valid hunt as a non antagonist scientist.

Non-antagonists have tools as their disposal to initiate a valid hunt from the start (toolboxes as weapons, stun prods for stuns, fire extinguishers for slip etc) . This does NOT mean that the non-antagonist is allowed to valid hunt. 
I don't like your mindset because it would coincide with the removal of stun prods or toolboxes from this game.


> Xenobiology is still absolutely as big a deal as people make it out to be. Don't try to take things piecemeal out of xenobiology-either we address it whole, or not at all. We aren't 'just' addressing xenolings. We're addressing xenobiology balance, of which xenolings are a part.

I stand by my statements. **In this PR you are not addressing the balance of xeno as a department. You are removing a feature from our game.** We would be having a different exchange if your proposed changes were to change the way you obtain these features or change the way the feature works itself.

as an FYI at the time of me writing this reply Pheromone receptors have been removed.
This was a big part in "valid hunting ability". I still think we should remove lingchat from the equation but that is a topic for a future PR. We are addressing you wanting to remove this feature as a whole.

> Would xenolings, on their own, be balanced? Probably not, but it would leave room for argument. However, xenolings with access to mob armies, the ability to turn into rare golem species, timestop, and instant adminheal cores are a different matter entirely, which is why we brought it up. 

**Why is having a non validhunting, non antagonist crew member with these abilities a bad thing?**
This is the main problem I have with your argument it relies on rule breaking to be valid.
I would like to point out that we should be making features in mindset of our main server rules being followed. 

This is going to be a bad example but if we are talking about balancing for rule breaking lets talk about flashbangs and stun batons for security. 
A security officer can throw a flashbang in to a bar, stun everyone inside and start killing them one by one. An officer can also use these tools to hijack the shuttle as well or prevent all greentexts by killing everyone on-shuttle as well. 
The reason these things are not brought up during flashbang and stun baton balance talks is that it is counter-intuitive to talk about a use-case for a item that is against our server rules. A security Officer is not allowed to kill our crew or hijack our shuttle. **Why should we remove a feature or balance an item because a non-antagonist scientist is breaking server rules to valid hunt?**

>Don't try to separate it like 'well this PR balances xenolings' because everything else is why this is a discussion at all.

**Your PR specifically targets xenolings as a feature.** 
I am going to focus on this part because that is all your PR changes at the moment. 
If you want to talk about xeno as a whole we need to propose to change aspects of xeno as a whole instead of laser focusing on xenolings. 
In my opinion, Xenolings do not define the xeno department at this time so we will be focusing on the changes you are proposing.

> Robotics and especially botany also need very little time to become problem children. But unless your shift ends in thirty minutes or less (IE: Wizard) xenobiologists usually have enough time to field what they want from their department.

I fail to see your point in this statement. Can you clarify?
You are saying that Xeno is able to field what they want from their department given enough time like Robotics and Botany but I am unsure how this supports your argument. 
Every department is able to field things from their department given enough time. The cook is able to field a whole pizza if you give him a few minutes.

> Xenoling needs merely a gold slime and a rainbow slime. This is achievable in twenty minutes. A xenobiologist can become a changeling in less time than it takes for war ops to even arrive.

I do not agree with this. I respectfully want to deny this. 
You are blowing it out of proportion please see the existing comments in this PR from experienced xeno users about the amount of time it takes to get a better understanding of the time aspect we are discussing.

> There's ample evidence for shifts where it DID matter that the xenobiologist became a changeling. Just look on the forums. For christ sake, I can't even play and even I can see that.

**Can you provide an example that isn't the xenobiologist breaking a server rule?**
I will turn around my stance if you can base your argument on xeno being an issue if server rules are followed.

 > Any round that goes on for longer than 40 minutes is virtually guaranteed to have a friendly crewmember running around who is immune to any and all damage,

xenolings are not immune to damage.
They get killed similar to a changeling. changelings are not unstoppable.
Again, **the main disconnect we are having is that I do not see how a non-antagonist , non valid hunting crew member is a big deal that warrants the removal of a comprehensive feature.**

> It merely rewards powergaming. There isn't any reason to have it, not even as a reward.

In your subjective opinion. Rewards are there in every department in order to reward time spent. 
Similar to how mining has powerful items, botany has strains that you can mix in to your plants, robotics has powerful mechs they can field. 
All of the things I have mentioned reward a player for taking the time to do things in their department. **We are playing a video game and getting rewards is fun.**

> 'From the perspective of being the crew, I don't see why any of these are bad'
> 
> Really? You don't see why on the fly, no-downsides stun immunity, rapid damage healing, and literal immunity to being killed is bad?

Yes, Adam. 
That is why I stated "I don't see why any of these are bad". 
Is this point trying to insult my point or do you actually want me to reinforce what I think?

File: code/modules/mob/living/simple_animal/hostile/headcrab.dm (hunk 1 above)
Comment (ThatLing) [reply code/modules/mob/living/simple_animal/hostile/headcrab.dm:25]: It's fine dude, jesus

**Comment by ghost** [comment]
@MonsWTF 

It’s not horseshit. I’ve pulled it off before. You need red slimes. That’s pretty easy.

@dr.derp3635

I’m moving onto nanites after this PR too. Of course, nanites have an actual counter, so i won’t be flat removing them, just making them more tolerable and in line with everything else when it comes to stun resistance and so forth.

I might even add a nanite injector for traitor scientists. That might be fun, but i’m just spitballing. Point is, don’t argue with something I already plan to smash with a hammer after the fact.

**Comment by ghost** [comment]
@Hopekz

Responding to you is a lost cause. Did you read a single thing written?

If I need to explain to you why nonantag crewmembers being able to turn themselves into changelings, or antagonists that are balanced around not being indestructible making themselves into lings being able to do exactly that, then there’s no point having this ‘discussion’.

You can have your opinion, and i’ll be right.

**Comment by alexkar598** [comment]
I mean TBH there isn't really a point in discussing because at the end,council decides the fate of this PR but I guess thumbs up and thumbs down might have an impact on their decision

**Comment by Amelia0010** [comment]
It's a server vote not a council vote :)

**Comment by MonsWTF** [comment]
> It’s not horseshit. I’ve pulled it off before. You need red slimes. That’s pretty easy.

You can keep saying it's easy, and I'll keep telling you you're full of it. Nothing productive or worthwhile is gained from lying to push an agenda.

**Comment by ghost** [comment]
@MonsWTF 

I am not lying. I've pulled it off before, so I know it's possible.

**Comment by Hopekz** [comment]
> @Hopekz
> 
> Responding to you is a lost cause. 

Are you giving up already? 
You said you were going to keep insulting me until you "win" but you've given up pretty quick..
Didn't you say that you had a leg to stand on with your **weak** argument?

>Did you read a single thing written?

Yes, Adam. 
Not only did I read what you wrote, **I fully responded to your post and even provided you examples as to why I think you have a weak argument**.

> If I need to explain to you why nonantag crewmembers being able to turn themselves into changelings, or antagonists that are balanced around not being indestructible making themselves into lings being able to do exactly that, then there’s no point having this ‘discussion’.

**Adam please, read the previous response**. 
I went in to detail about how this is only an issue if you expect individuals to not follow our server rules. 
Validhunting is not allowed on our server.
Not following the rules is the reason you are currently banned from our server. 
The playstyle in mind that you are trying to cater to of valid hunting as a non antag scientist is idiotic and does not belong on here.
You should know more than anyone that we ban for breaking our rules as you are banned yourself.

> You can have your opinion, and i’ll be right.

**If you don't provide anything more than "I'm right hah" then you have lost this exchange.** 
My opinion remains unchanged as I was hoping that your weak argument for this change had more to it.
**Please stop introducing meme PR's and expect people to take you seriously if you can't even back up your own points.**

You still have the option to answer my previous points but if you reply with "hah I'm right responding is stupid because I'm right" our exchange is over. 
I am here to collaborate.

Side point:
Unsure why we are taking this meme PR seriously especially if the author isn't willing to exchange.
-1 to this PR.

**Comment by ghost** [comment]
thanks for your opinion dude but like if it isn't inherently obvious to you why crewmembers being able t make themselves able to revive from death automatically then that's your problem and one i can't explain to you

pretend you're here to collaborate all you want you haven't made any actual points i can respond to 

it's literally just a repeat of 'but VALIDHUNTING is BAD and NOT ALLOWED' and that's your literal only argument

like how do i explain to you that _bannable_ validhunting isn't the issue? yeah if some dude rushes down the halls with fleshmend looking for antags, that gets him banned as a scientist

it doesn't get the sec officer banned who does it, it doesn't get the guy who makes himself a xenoling on clock cult or war ops banned, it doesn't get you banned if a ling actively attacks you and you just so happen to be completely immortal and have access to their comms, it doesn't get you banned if a traitor ambushes you and you're fucking immortal, etc

the problem isn't a rules issue and people abusing it to validhunt in a way that gets them banned it's a balance issue as it makes people unfairly strong in a way that most antagonists can't counter effectively because it's often an unpleasant surprise that the guy you just attacked just so happens to be a changeling much like it's often an unpleasant surprise that the guy you just attacked just so happens to be packing a disabler and a syringe gun full of zombie powder and a meth-healmix pill on hand

the difference is that one gets you banned for powergame but the other is someone having access to shit they shouldn't have, for no penalty

if you order SWAT gear as cargo and arm up cargo in the armor, just the armor, not the guns, 'just in case' you're probably gonna get in trouble

if you make yourself a changeling, you're even harder to kill than that but get out scott free and worse, there's no indication you are, if you just so happen to be out of xeno. or get made into a ling and then get a job change to sec. or if you make someone from sec a ling. etc. 

it's a matter of unfair abilities and upsetting antagonist balance by means of making people indestructible. you can literally come back as many times as you want so long as you aren't cremated, gibbed, or absorbed, and those are particularly unlikely to happen during round one where the guy isn't dragging you to the crematorium for you to pop up and give him an ass whooping in round two

if all of that isn't inherently apparent to you-if it isn't somehow clear to you that non-antagonist crew should _not_ have access to the means to automatically heal off massive severe damage (including death) without even looking at medbay, the means to eavesdrop on changelings and swap species on a dime, and so on and so forth with stun resistance to make it impossible to stun them effectively and speedboosts to make them hard to catch- if it isn't somehow automatically clear when i say that on the fly, no-downsides stun immunity, rapid damage healing, and literal immunity to being killed is bad that these are the reasons why, then i have no clue what i could possibly say to you because you're clearly on an entirely different wavelength.

even if we discard all the arguments about nonantags, this still isn't even taking into consideration antags getting it.

note that none of the situations i described above-literally none of them-involve validhunting at all. give it a read. go on. if you say 'but the rules prohibit it' as an argument after this you lose.

**Comment by Hopekz** [comment]
**Edit 10/30/2019**: It looks like the comment I'm replying to in this post doesn't exist anymore... good thing I included what was said in my reply. 
This is a reply to Adam's statement.

---
> pretend you're here to collaborate all you want you haven't made any actual points i can respond to

Hello Adam,
Here are my questions that I'm bringing down from my previous responses. 
Its a little messy so you might have an easier time just replying to those original arguments instead.
I am not modifying them in any way except making them easier to see after just copy-pasting them from my previous messages.

**Why is taking down a non-antag that big of a deal?**
---
**Are we dealing with more of a self antag issue with these changes?**
---
In your robotics point this question was brought up:
**Why would you remove the Durand from the game just because robotics can take the time to build it?** 
---
**Before replying to this one please look back at that part in the argument (ctrl + f) as context will be important.**

Why is having a non validhunting, non antagonist crew member with these abilities a bad thing?
-----
Why should we remove a feature or balance an item because a non-antagonist scientist is breaking server rules to valid hunt?
-----

>Robotics and especially botany also need very little time to become problem children. But unless your shift ends in thirty minutes or less (IE: Wizard) xenobiologists usually have enough time to field what they want from their department.

I fail to see your point in the above statement. Can you clarify? 
---
**I provided you an explanation on why it was confusing for more context(ctrl + f)**



>There's ample evidence for shifts where it DID matter that the xenobiologist became a changeling. Just look on the forums. For christ sake, I can't even play and even I can see that.

from your above point: Can you provide an example that isn't the xenobiologist breaking a server rule?
-----


> it's literally just a repeat of 'but VALIDHUNTING is BAD and NOT ALLOWED' and that's your literal only argument

Not my only argument but see the following for why this is important:
Copy pasting from previous response::
"
This is going to be a bad example but if we are talking about balancing for rule breaking lets talk about flashbangs and stun batons for security.
A security officer can throw a flashbang in to a bar, stun everyone inside and start killing them one by one. An officer can also use these tools to hijack the shuttle as well or prevent all greentexts by killing everyone on-shuttle as well.
The reason these things are not brought up during flashbang and stun baton balance talks is that it is counter-intuitive to talk about a use-case for a item that is against our server rules. A security Officer is not allowed to kill our crew or hijack our shuttle. **Why should we remove a feature or balance an item because a non-antagonist scientist is breaking server rules to valid hunt?**
"
( I even went ahead and highlighted one of those questions you were trying to find)

**Comment by deadmomo69** [comment]
> This is going to be a bad example but if we are talking about balancing for rule breaking lets talk about flashbangs and stun batons for security.

if its a bad example perhaps you should choose a better one that can be compared to xenolings

> A security officer can throw a flashbang in to a bar, stun everyone inside and start killing them one by one. An officer can also use these tools to hijack the shuttle as well or prevent all greentexts by killing everyone on-shuttle as well.
> The reason these things are not brought up during flashbang and stun baton balance talks is that it is counter-intuitive to talk about a use-case for a item that is against our server rules. A security Officer is not allowed to kill our crew or hijack our shuttle. Why should we remove a feature or balance an item because a non-antagonist scientist is breaking server rules to valid hunt?

what is the feature behind xenolings, other than to validhunt should be the question..
the only applicable feature is 'never die' because of regenerative stasis, unless dusted/cremated etc.

securitys feature is to be security, the intent behind their flashbangs/stunbatons were never to muderbone.

**Comment by Hopekz** [comment]
> if its a bad example perhaps you should choose a better one that can be compared to xenolings

I find it difficult to make direct examples for this as I think xenolings is its own feature that doesn't really have any direct comparisons on the non-antag crew side. 

> what is the feature behind xenolings, other than to validhunt should be the question..

I would think it would be a reward for playing their own department for that amount of time. 
I like to think of this reward being in-line with for example getting a stand + good lava land loot as non antag miner. Having these items as a reward even to a miner allows them to also perform the same valid hunt actions that this PR is trying to counter for xeno.

I think miners do not really "need" these items similar to how xeno does really "need" xenolings to function but it is nice having it there as a gameplay feature.

> the only applicable feature is 'never die' because of regenerative stasis, unless dusted/cremated etc.

Let us also remember that this PR is attempting to remove the feature as a whole from the game.
I am perfectly fine with discussing how this feature should be modified to fit in to the game better. 

> securitys feature is to be security, the intent behind their flashbangs/stunbatons were never to muderbone.

I agree with you on this. 
My example wasn't really too good but I'm mainly trying to get the idea of items being balanced for their use-case across. I do not think whole features should be removed with the main reason being that an offending individual has the ability to go against established rules.

my main point is that removing a comprehensive feature from a code-base because of this reasoning is kind of counter-intuitive to having a game with features. 
Kind of reminds me of Koji's "remove" mentality. 

I want to be able to play a game where I am able to perform these things. 
I think we should work to change the way this feature works instead of wiping it out as currently proposed by this PR.

**Comment by ghost** [comment]
i answered your questions in the above post. all of them. read it again. nothing about validhunting in here. it's balance.

**Comment by ShadowDeath6** [comment]
I preferred xenolings when rainbow slime mindswap potion didn’t exist

**Comment by LoliconSlayer** [comment]
Imagine thinking letting people become immortal with a shit ton of dumb as fuck powers through """effort""" is a good thing. Improve not remove definitely rings true here except the thing being improved is the game as a whole by removing the xenolings.

You might be permabanned Adam, but props to you for doing God's work

**Comment by Moja10** [comment]
here's my perspective as I view it as an admin: I rarely see xenolings for obvious reasons, it is a significant task to make one at all, but they do cause problems and I rarely see rounds improved by their existence. I do see the effort in acquiring xenoling, but it's not worth the shittiness they bring in a round for everyone involved except the one dude who jacked off in xenobio for an hour. Personally I don't think it's quite comparable to things other departments get because none of them get straight up antag powers. Adam is right about this, it's always been an elephant in the room complete nonsense that contrasts drastically with every other part of the station, which is a fair comparison to make. 

There's already been a PR about this, a long time ago, because they were causing problems in the past with having antag nonsense on a grindy role. PR removed majority of the direct antag nonsense they had, and it still wasn't enough.  They were given I believe more than a year of existence in their current state, and despite being rare, still caused problems and created nonsense rounds with their existence. 

If I could remove them over anything its because I don't think the game is about afking until antag powers. Fundamentally against the design philosophy of this game. Doesn't mesh with any other existing balance structures that exist.

**Comment by MegaEmpirical** [comment]
I keep seeing it on low pop at night more than high pop/day.

**Comment by Nickvr628** [comment]
This is dumb, I hope this gets closed. Just because something has potential to be antagonistic for non-antags does not mean it needs to be removed entirely. 

Might as well delete combat mechs, volume pumps, hulk, and toxins while you are at it.

**Comment by Amelia0010** [comment]
stop with the whataboutisms.

**Comment by ghost** [comment]
nick thank you for confirming every negative thought i have ever had about you every time you open your mouth

**Comment by AsV9** [comment]
https://forums.yogstation.net/index.php?threads/vote-merge-6926.20027/

**Comment by Mr-Space** [comment]
ladies and gentleman, we have made history today

**Comment by oklews** [comment]
BASED

**Pull request merged**